recipes/libs/qt6-base.yaml

```
@@ -6,6 +6,55 @@ metaEnvironment:
         GPL-2.0+ or LGPL-3.0, GPL-3.0 with exception (tools), GFDL-1.3 (docs),
         Apache-2.0, BSD-3-Clause, BSL-1.0, MIT
 
```

How a about declaring the option with a proper type and setting a default?

```suggestion
Config:
    QT6_BASE_HOST_BUILD:
        type: bool
        default: False
        help: "Host tools build with reduced dependencies"
```

---

done